unlock-app/src/components/creator/lock/LockIconBar.js

```diff
@@ -1,22 +1,31 @@
 import React from 'react'
 import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
 import UnlockPropTypes from '../../../propTypes'
 import Buttons from '../../interface/buttons/lock'
 
-export function LockIconBar({ lock }) {
-  return (
-    <IconBar>
-      <Buttons.PreviewLock lock={lock} />
-      <Buttons.Withdraw />
-      <Buttons.Edit />
-      { /* Reinstate when we're ready <Buttons.ExportLock /> */ }
-      <Buttons.Code />
-    </IconBar>
-  )
+export class LockIconBar extends React.Component {
```

Comment:
Why move to a full `React.Component` ? it looks like the stateless function should still work?